<launch>
    <node name="tf_server" pkg="human_baxter_servers" type="tf_server" output="screen"/>
    <node name="moveit_controller_slow" pkg="human_baxter_servers" type="moveit_controller_slow" output="screen"/>
</launch>
